src/main/java/kitchenpos/order/domain/Order.java

```diff
@@ -0,0 +1,103 @@
+package kitchenpos.order.domain;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Long orderTableId;
+    private String orderStatus;
```

Comment:
```suggestion
    private OrderStatus orderStatus;
```
enumerated 어노테이션을 사용해서 문자열이 아닌 enum으로 처리하도록 수정해보아요